### DISCUSSION TITLE
Subir projeto C# completo para o Github

### DISCUSSION THREAD

**Discussion by ReixVictor** [community/community#70172] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Projects

### Body

Estou enfrentando o seguinte problema: estou subindo um projeto do Visual Studio para o GitHub, e sempre estou atualizando, mas quando vou clonar o repositorio em outro PC, o projeto parece vir "incompleto", com diversos tipos de erro.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @ReixVictor , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Additionally, you are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects.
